python/lsst/ts/standardscripts/maintel/prepare_for/align.py

```diff
@@ -0,0 +1,124 @@
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["PrepareForAlign"]
+
+import yaml
+from lsst.ts.observatory.control.maintel.mtcs import MTCS, MTCSUsages
+
+from lsst.ts import salobj
+
+
+class PrepareForAlign(salobj.BaseScript):
+    """Run prepare for align on MTCS.
+
+    Parameters
+    ----------
+    index : `int`
+        Index of Script SAL component.
+
+    Notes
+    -----
+    **Checkpoints**
+
+    "Positioning telescope.": Set position at which to perform alignment.
+    "Closing mirror covers.": Check mirror covers are closed.
+    "Checking components...": Check all components have the right LUT.
+    "MTCS Ready for...": MTCS is ready to start alignment.
+
+    """
+
+    def __init__(self, index: int, add_remotes: bool = True):
+        super().__init__(index=index, descr="Prepare for MTCS laser tracker alignment.")
+
+        self.config = None
+
+        self.mtcs = MTCS(
+            domain=self.domain,
+            intended_usage=None if add_remotes else MTCSUsages.DryTest,
+            log=self.log,
+        )
+
+    @classmethod
+    def get_schema(cls):
+        url = "https://github.com/lsst-ts/"
+        path = (
+            "ts_externalscripts/blob/main/python/lsst/ts/standardscripts/"
+            "maintel/prepare_for/align.py"
+        )
+        schema_yaml = f"""
+        $schema: http://json-schema.org/draft-07/schema#
+        $id: {url}{path}
+        title: PrepareForAlign v1
+        description: Configuration for Prepare for alignment with laser tracker.
+        type: object
+        properties:
+            tel_align_az:
+                description: telescope azimuth angle at which alignment will be performed in deg.
+                type: number
+                default: 0.0
+                minimum: 0.0
+                maximum: 360.0
+            tel_align_el:
+                description: telescope elevation angle at which alignment will be performed in deg.
+                type: number
+                default: 60.0
+                minimum: 16.0
+                maximum: 86.0
+            tel_align_rot:
+                description: telescope rotation angle at which alignment will be performed in deg.
+                type: number
+                default: 0.0
+                minimum: 0.0
+                maximum: 360.0
+        additionalProperties: false
+        """
+        return yaml.safe_load(schema_yaml)
+
+    async def configure(self, config):
+        self.tel_align_az = config.tel_align_az
+        self.tel_align_el = config.tel_align_el
+        self.tel_align_rot = config.tel_align_rot
+
+    def set_metadata(self, metadata):
+        metadata.duration = 600.0
+
+    async def run(self):
+        """Run the script."""
+
+        await self.checkpoint("Positioning Telescope.")
+        await self.mtcs.point_azel(
```

Comment:
I think moving the telescope should probably be the last thing done by the script..